packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -3,31 +3,75 @@ import $ from '@js/core/renderer';
 import type { Properties } from '@js/ui/chat';
 
 import Widget from '../widget';
+import ChatHeader from './chat_header';
+import MessageBox from './chat_message_box';
+import MessageList from './chat_message_list';
 
 const CHAT_CLASS = 'dx-chat';
 
+const MOCK_CURRENT_USER_ID = 'CURRENT_USER_ID';
+
 class Chat extends Widget<Properties> {
+  _chatHeader?: ChatHeader;
+
+  _messageBox?: MessageBox;
+
+  _messageList?: MessageList;
+
   _getDefaultOptions(): Properties {
     return {
       ...super._getDefaultOptions(),
-      ...{
-        items: [],
-        onMessageSend: undefined,
-      },
+      title: '',
+      items: [],
+      onMessageSend: undefined,
     };
   }
 
   _initMarkup(): void {
     $(this.element()).addClass(CHAT_CLASS);
 
     super._initMarkup();
+
+    this._renderHeader();
+    this._renderMessageList();
+    this._renderMessageBox();
+  }
+
+  _renderHeader(): void {
+    const { title } = this.option();
+
+    const $header = $('<div>').appendTo(this.element());
+
+    this._chatHeader = this._createComponent($header, ChatHeader, { title: title ?? '' });
```

Comment:
Title default value is ''. Why we should process undefined value here?

Reply:
[74da06c994b50425c1d5a62c53df51ab7f9f9438]